docker-compose.yaml

```diff
@@ -70,3 +70,20 @@ services:
         condition: on-failure
         delay: 20s
         max_attempts: 3
+        
+  quest-load:
+    image: ghcr.io/parseablehq/quest:main
+    command: ["load", "http://parseable:8000", "parseableadmin", "parseableadmin", "20"]
+    networks:
+      - parseable-internal
+    depends_on:
+      quest-smoke: 
+        condition: service_completed_successfully
+      - parseable
```

Comment:
This doesn't look like it is needed here?

Reply:
are you talking about paraseable

Reply:
should I remove this.

Reply:
should I remove this.

Reply:
yes, please remove

Reply:
naming issues occur. What name should I give now?